### DISCUSSION TITLE
Risolto - Lampadina Ledvance zigbee + ha+zigbee2MQTT.

### DISCUSSION THREAD

**Discussion by AleDestroyer** [indomus/forum#3203] [GENERALE - Discussioni generiche]
Ciao a tutti,
ho acquistato una lampadina ledvance zigbee, riconosciuta ed installata senza problemi da ha+zigbee2MQTT.
Ho un problema quando spengo la lampadina dall'interruttore di casa, lo stato non viene aggiornato e su homeassistant rimane lo stato su on/accesa.
Avete avuto anche voi quest problema? Consigli su come risolvere?

Grazie mille :)

**Comment by KC-inDomus** [comment]
Naturalmente si tratta di un comportamento naturale: la lampadina, disalimentandola, non è off ma unresponsive. Le lampadine zigbee non sono fatte per essere spente elettricamente, ma solo a livello logico.

**Reply by AleDestroyer** [reply]
Ciao, 
infatti, non so perchè ma ero convinto funzionasse anche ad interruttore spento, diciamo che per l'uso che ne faccio io sono meglio gli switch.  

Grazie mille
